core/src/main/java/com/googlecode/psiprobe/tools/url/UrlParser.java

```diff
@@ -34,6 +34,13 @@
 
   /**
    * Instantiates a new url parser.
+   */
+  public UrlParser() {
```

Comment:
Yeah, the vastly increased test coverage is a welcome sight! We'll get this ironed out. Thanks for being open to revisiting these changes.

I don't think anyone is extending it or depending on it. The only intended client for its core API is the web app, and then much of the API doesn't work outside the context of the PSI Probe web app running on a supported server. Out of curiosity, why do you ask?